### ISSUE TITLE
"show all" / "bugs per history tab"

### ISSUE THREAD

**Issue by jfriedel** [OpenGrok/OpenGrok#1712]
Some history logs are long (many entries / changesets), and OpenGrok divides them into pages (with certain number of bugs on a single page). If two entries in the history end up on two distinct pages, there is no way how one could compare them (the entries / changesets). Either a drop-down box with entries per page or "show all" button (so all entries are shown on a single page) would be useful. Of course, you're mileage may vary and a better solution can be found..

**Comment by vladak** [comment]
drop down box or slider with number of items per page would work for me.

More sophisticated solution would be some search box for history entries.

**Comment by vladak** [comment]
This is related to #1049

**Comment by tulinkry** [comment]
> If two entries in the history end up on two distinct pages, there is no way how one could compare them

This is not true, it should be working that you can select two entries on different pages.

>  Either a drop-down box with entries per page...

Current workaround could be adding a `n` parameter to the url to display greater number of entries.

**Comment by jfriedel** [comment]
Correct; I was wrong about the selection. Feel free to close this issue as not a bug if the `n' parameter is not worth to implement. Thanks ;)

**Comment by vladak** [comment]
I think it's still worthwhile addition - at least for me. I often bring up the history of a file and then search the contents for keywords as opposed to searching the history using the search.
